week-5/ghostdb/Scripts/1_script_create_tables.sql

```diff
@@ -0,0 +1,40 @@
+create TABLE customers(
+  customer_id serial PRIMARY KEY,
+  customer_name text,
+  customer_city text,
+  customer_state text,
+  customer_country text
+);
+
+create TABLE products(
+	product_id serial PRIMARY KEY,
+	product_name  text,
+	days integer,
+	price integer
+);
+
+create TABLE sales(
+  sale_id serial PRIMARY KEY,
+  product_id integer REFERENCES products(product_id),
+  customer_id integer REFERENCES customers(customer_id),
+  transaction_date date,
+  payment_type text,
+  sold_by text,
+  contact text,
+  pos text
+);
+
+create TABLE temporal(
+  transaction_date date,
+  product_name  text,
+  days integer,
+  price integer,
```

Comment:
How about if your price is 1200.00?

Reply:
Good observation, but before we used integer since we saw all kind of value in the file for that coumn, so with purpose of working with values easier, we did it.

Reply:
Oh, I got it. So it's ok for me.

Reply:
@deyviconde  SQL just shows an error.

Reply:
What do you mean ? 👀

Reply:
SQL won't accept it.